### PR TITLE
Update deploy-enterprise.yml to increase product lifetime to 2 months

### PR DESCRIPTION
**Pull request by cmarshak**
Just want to be extra sure things stay in Tibet account for at least 2 months.

### CODE REVIEW

**Review by jtherrmann — APPROVED** [review]

**Comment by jtherrmann** [comment]
Hi @cmarshak, this change looks fine to me, but normally I would wait for @asjohnston-asf's approval before releasing changes to any of our JPL deployments, as I don't have access to the AWS console for those deployments. Are you fine with waiting until @asjohnston-asf is back in the office next week?

**Comment by cmarshak** [comment]
Yes - no problem - I forgot he was out :)

**Review by asjohnston-asf — APPROVED** [review]

**Pull request merged**
